hrjobcontract/js/src/job-contract/controllers/form/form-pay.controller.js

```diff
@@ -28,7 +28,6 @@ define(function () {
       Hour: 2080
     };
 
-    $scope.collapseBenDed = !entityPay.annual_benefits.length && !entityPay.annual_deductions.length;
```

Comment:
I would not actually move $scope to init(). I would rather suggest to check for the prop to exist:

```js
$scope.collapseBenDed =
  (!entityPay.annual_benefits || !entityPay.annual_benefits.length) &&
  (!entityPay.annual_deductions || !entityPay.annual_deductions.length);
```